code/modules/mob/dead/crew_manifest.dm

```diff
@@ -0,0 +1,34 @@
+GLOBAL_DATUM_INIT(crew_manifest_tgui, /datum/crew_manifest, new)
+
+/datum/crew_manifest
+
+/datum/crew_manifest/ui_state(mob/user)
+	return GLOB.always_state
+
+/datum/crew_manifest/ui_status(mob/user, datum/ui_state/state)
+	var/static/list/allowed_mobs_typecache = typecacheof(list(/mob/dead, /mob/living/silicon))
+	return is_type_in_typecache(user, allowed_mobs_typecache) ? UI_INTERACTIVE : UI_CLOSE
+
+/datum/crew_manifest/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "CrewManifest")
+		ui.set_autoupdate(TRUE)
+		ui.open()
+
+/datum/crew_manifest/ui_static_data(mob/user)
+	return list(
+		"command" = list(
+			"huds" = GLOB.command_huds,
+			"jobs" = GLOB.command_positions,
+			"order" = SSjob.chain_of_command
+		),
+		"order" = list_to_assoc_index(flatten_list(GLOB.id_to_hud)),
```

Comment:
static data, contrary to popular belief, is not static. this is kinda wasteful to convert over and over